<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>553</width>
    <height>284</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Ubuntu</family>
   </font>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>0</y>
     <width>281</width>
     <height>281</height>
    </rect>
   </property>
   <widget class="QWidget" name="login">
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>221</width>
       <height>51</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit#lineEdit {
  font-family: &quot;Microsoft YaHei&quot;;
   font-size: 18px;
   color: rgb(50, 50, 50);
	background-color: rgba(255, 255, 255, 0%);
	border:2px solid #b9babb;
    border-radius: 4px;
    padding: 2px 2px;
    min-height: 18px;   /* 对应高度大概是24px，这里不知为何不能等价 */
    padding-right: 18px;
}
QLineEdit#lineEdit:focus{
    border: 2px solid #014099;
}
 
QLineEdit#lineEdit:hover{
    border: 2px solid #014099;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>221</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit#lineEdit_2 {
   font-family: &quot;Microsoft YaHei&quot;;
   font-size: 18px;
   color: rgb(50, 50, 50);
	background-color: rgba(255, 255, 255, 0%);
	border:2px solid #b9babb;
    border-radius: 4px;
    padding: 2px 2px;
    min-height: 18px;   /* 对应高度大概是24px，这里不知为何不能等价 */
    padding-right: 18px;
}
QLineEdit#lineEdit_2:focus{
    border: 2px solid #014099;
}
 
QLineEdit#lineEdit_2:hover{
    border: 2px solid #014099;
}
QLineEdit#lineEdit_2 QPushButton {
    width:  16px;
    height: 16px;
    qproperty-flat: true;
    margin-right: 4px;
    border: none;
    border-width: 0;
    background: transparent;
    border-image: url(:/image/hide_password.png) 0 0 0 0 stretch stretch;

}
 
QLineEdit#lineEdit_2 QPushButton::checked {
 	border-image: url(:/image/show_password.png) 0 0 0 0 stretch stretch;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>170</y>
       <width>221</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#pushButton
{
border: none;
color: white;
padding: 15px 32px;
text-align: center;
border-radius: 20px;
text-decoration: none;
background-color: rgb(0,229,238);
}
QPushButton#pushButton:hover
{
border: none;
color: white;
padding: 15px 32px;
text-align: center;
border-radius: 20px;
text-decoration: none;
background-color: rgb(142,229,238);
}
QPushButton#pushButton:pressed
{
border: none;
color: white;
padding: 15px 32px;
text-align: center;
border-radius: 20px;
text-decoration: none;
background-color: rgb(85,170,255);
}</string>
     </property>
     <property name="text">
      <string>登录</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>260</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#pushButton_2
{
  border-style:none none solid none;
}
QPushButton#pushButton_2:hover
{
  border-width:1px;
  border-style:none none solid none;
  color:  rgb(239, 41, 41);
  border-color:#1093cb;
}
QPushButton#pushButton_2:pressed
{
  border-width:1px;
  border-style:none none solid none;
  color: rgb(204, 0, 0);
  border-color:#1093cb;
}
</string>
     </property>
     <property name="text">
      <string>没有账号？注册一个</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="register_2">
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>221</width>
       <height>51</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit#lineEdit_3 {
  font-family: &quot;Microsoft YaHei&quot;;
   font-size: 18px;
   color: rgb(50, 50, 50);
	background-color: rgba(255, 255, 255, 0%);
	border:2px solid #b9babb;
    border-radius: 4px;
    padding: 2px 2px;
    min-height: 18px;   /* 对应高度大概是24px，这里不知为何不能等价 */
    padding-right: 18px;
}
QLineEdit#lineEdit_3:focus{
    border: 2px solid #014099;
}
 
QLineEdit#lineEdit_3:hover{
    border: 2px solid #014099;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>221</width>
       <height>51</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit#lineEdit_4 {
   font-family: &quot;Microsoft YaHei&quot;;
   font-size: 18px;
   color: rgb(50, 50, 50);
	background-color: rgba(255, 255, 255, 0%);
	border:2px solid #b9babb;
    border-radius: 4px;
    padding: 2px 2px;
    min-height: 18px;   /* 对应高度大概是24px，这里不知为何不能等价 */
    padding-right: 18px;
}
QLineEdit#lineEdit_4:focus{
    border: 2px solid #014099;
}
 
QLineEdit#lineEdit_4:hover{
    border: 2px solid #014099;
}
QLineEdit#lineEdit_4 QPushButton {
    width:  16px;
    height: 16px;
    qproperty-flat: true;
    margin-right: 4px;
    border: none;
    border-width: 0;
    background: transparent;
    border-image: url(:/image/hide_password.png) 0 0 0 0 stretch stretch;

}
 
QLineEdit#lineEdit_4 QPushButton::checked {
 	border-image: url(:/image/show_password.png) 0 0 0 0 stretch stretch;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>130</y>
       <width>221</width>
       <height>51</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit#lineEdit_5 {
   font-family: &quot;Microsoft YaHei&quot;;
   font-size: 18px;
   color: rgb(50, 50, 50);
	background-color: rgba(255, 255, 255, 0%);
	border:2px solid #b9babb;
    border-radius: 4px;
    padding: 2px 2px;
    min-height: 18px;   /* 对应高度大概是24px，这里不知为何不能等价 */
    padding-right: 18px;
}
QLineEdit#lineEdit_5:focus{
    border: 2px solid #014099;
}
 
QLineEdit#lineEdit_5:hover{
    border: 2px solid #014099;
}
QLineEdit#lineEdit_5 QPushButton {
    width:  16px;
    height: 16px;
    qproperty-flat: true;
    margin-right: 4px;
    border: none;
    border-width: 0;
    background: transparent;
    border-image: url(:/image/hide_password.png) 0 0 0 0 stretch stretch;

}
 
QLineEdit#lineEdit_5 QPushButton::checked {
 	border-image: url(:/image/show_password.png) 0 0 0 0 stretch stretch;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>200</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#pushButton_3
{
border: none;
color: white;
padding: 15px 32px;
text-align: center;
border-radius: 20px;
text-decoration: none;
background-color: rgb(0,229,238);
}
QPushButton#pushButton_3:hover
{
border: none;
color: white;
padding: 15px 32px;
text-align: center;
border-radius: 20px;
text-decoration: none;
background-color: rgb(142,229,238);
}
QPushButton#pushButton_3:pressed
{
border: none;
color: white;
padding: 15px 32px;
text-align: center;
border-radius: 20px;
text-decoration: none;
background-color: rgb(85,170,255);
}</string>
     </property>
     <property name="text">
      <string>注册</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>200</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#pushButton_4
{
border: none;
color: white;
font-size:32px;
padding: 15px 32px;
text-align: center;
border-radius: 20px;
text-decoration: none;
background-color: rgb(0,229,238);
}
QPushButton#pushButton_4:hover
{
border: none;
color: white;
font-size:32px;
padding: 15px 32px;
text-align: center;
border-radius: 20px;
text-decoration: none;
background-color: rgb(142,229,238);
}
QPushButton#pushButton_4:pressed
{
border: none;
color: white;
font-size:32px;
padding: 15px 32px;
text-align: center;
border-radius: 20px;
text-decoration: none;
background-color: rgb(85,170,255);
}</string>
     </property>
     <property name="text">
      <string>←</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>221</width>
     <height>201</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
